### ISSUE TITLE
Update jQuery dependency

### ISSUE THREAD

**Issue by yashsway** [miguel-perez/smoothState.js#297]
There needs to be a note in the README to indicate that this only works with jQuery 2.2.4. This does not work with jQuery 3x. 

Also browsers now indicate localStorage changes which could affect the library:
'window.webkitStorageInfo' is deprecated. Please use 'navigator.webkitTemporaryStorage' or 'navigator.webkitPersistentStorage' instead.

**Comment by NewJenk** [comment]
Have you tried changing the ajax 'success, error and complete' callbacks to 'done, fail and always' as noted in issue #291?

**Comment by phifa** [comment]
is there a patch available for this? need to get it running with jquery 3 as well

**Comment by phifa** [comment]
yeah a patch would be great. someone has something ready? a pull request?

**Comment by puppetmaster3** [comment]
patch: https://github.com/miguel-perez/smoothState.js/issues/326
